src/host/ut_host/VtIoTests.cpp

```diff
@@ -51,9 +54,7 @@ static constexpr std::wstring_view s_initialContentVT{
 
 class ::Microsoft::Console::VirtualTerminal::VtIoTests
 {
-    BEGIN_TEST_CLASS(VtIoTests)
-        TEST_CLASS_PROPERTY(L"IsolationLevel", L"Class")

```

Comment:
Don't we want to maintain class isolation? This means that the test harness is fully cleared/reset when we enter and leave VtIotests.

Reply:
Is this why `ConPtyTests::DiesOnClose` is failing?

Reply:
Nope that’s in a separate module. That’s probably because of our shutdown changes right?

Reply:
It doesn't seem to occur in my other PRs nor locally, so it has me a bit confused why this specifically occurs on this PR.